dpgen2/entrypoint/submit.py

```diff
@@ -402,6 +404,82 @@ def submit_concurrent_learning(
     wf = Workflow(name="dpgen", context=lebesgue_context)
     wf.add(dpgen_step)
 
-    wf.submit()
+    wf.submit(reuse_step=reuse_step)
+
+    return wf
+
+
+def print_list_steps(
+        steps,
+):
+    ret = []
+    for idx,ii in enumerate(steps):
+        ret.append(f'{idx:8d}    {ii}')
+    return '\n'.join(ret)
+
+
+def expand_idx (in_list) :
+    ret = []
+    for ii in in_list :
+        if type(ii) == int :
```

Comment:
PEP 8:

> Object type comparisons should always use isinstance() instead of comparing types directly:
> 
> ```py
> # Correct:
> if isinstance(obj, int):
> ```
> 
> ```py
> # Wrong:
> if type(obj) is type(1):
> ```